src/App/Fossa/Config/Analyze.hs

```diff
@@ -103,13 +107,16 @@ import Options.Applicative (
   switch,
   (<|>),
  )
+import Options.Applicative.Builder (maybeReader)
 import Path (Abs, Dir, File, Path, Rel)
 import System.Info qualified as SysInfo
-import Types (TargetFilter)
+import Types (ArchiveUploadType (..), TargetFilter)
 
 -- CLI flags, for use with 'Data.Flag'
 data AllowNativeLicenseScan = AllowNativeLicenseScan deriving (Generic)
 data ForceVendoredDependencyRescans = ForceVendoredDependencyRescans deriving (Generic)
+data ForceVendoredDependencyCLILicenseScan = ForceVendoredDependencyCLILicenseScan deriving (Generic)
+data ForceVendoredDependencyArchiveUpload = ForceVendoredDependencyArchiveUpload deriving (Generic)
```

Comment:
I think these can be removed now, right?

Reply:
Oh right! Thank you